great-idea/index.html

```diff
@@ -17,6 +17,111 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+	  
+	   <header>
+ 	<section class="section1">
+	  <nav>
+ <a href="#">Services</a>
+  	<a href="#">Product</a>
+  	<a href="#">Features</a>
+	<a href="#">About</a>
+  	<a href="#">Contact</a>
+  	
+	</nav>
+  
+
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+	  </section>
+  
+	  <section class="section2">
+	  <h1 class="main-header"> 
+ 
+		  <div class="innovation">Innovation</div>    
+		<div class="on">On</div>
+		
+	  	   <div class="demand">Demand</div>
+		  
+		  <button>
+		Get Started  
+		  
+	</button> 
+		  
+		  </h1>
+
+ 	
+
+  <img src="img/header-img.png" alt="Image-of-a-code-snippet">
+		  </section>
+		  </header>
+<hr>
+	  
+	  <section class="section3">
+  <h3 class="features">
+	  Features
+	</h3>
+		  <h3 class="about">
+  About
+	</h3>
+	  </section>
+	 
+	  <section class="section4">
+  <p class="p-features">
+	Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </p>
+
+
+  <p class="p-about">
+	Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </p>
+	  </section>
+
+<section class="section5">
+  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+	  </section>
+	  
+<section class="section6">
+ <h3 class="services">
+	Services
+	 </h3>
+		  <h3 class="product">
+  Product
+	</h3>
+		  <h3 class="vision">
+	  Vision
+	</h3>
+	  </section>
+	  <section class="section7">
+  <p class="p-services">
+	Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </p>
+
+
+  <p class="p-product">
+	Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	  </p>
+
+  
+ <p class="p-vision">
+	Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+	 </p>
+	</section>
+	  <hr>
+	  <footer>
+<section class="section8">
+  <h3 class="contact">Contact</h3>
+	<address>
+	
+  123 Way 456 Street<br>
+  Somewhere, USA<br>
+  <div class= "phone">1 (888) 888-8888</div>
```

Comment:
`span` may be a better idea than `div` if you're nesting them inside `address`. The default display property for `span` is `inline`, and is more informative to other developers than `div`.